Builds/VisualStudio2015/RippleD.vcxproj

```diff
@@ -3353,6 +3353,10 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='debug|x64'">True</ExcludedFromBuild>
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='release|x64'">True</ExcludedFromBuild>
     </ClCompile>
+    <ClCompile Include="..\..\src\ripple\rpc\tests\AccountOffers.test.cpp">

```

Comment:
You don't need to do that. Simply run `scons vcxproj` and it will generate the Visual Studio project files correctly.